### ISSUE TITLE
Delete operation fails with strapi 3.0.1 stable and version 1.0.7 of the provider

### ISSUE THREAD

**Issue by alsmyr** [jakeFeldman/strapi-provider-upload-azure-storage#11]
Hi,

I integrated your provider in our strapi setup and upload works fine and the media ends up in blob storage as expected. However when I try to delete the operation fails with the logged error as below.

I just wanted to post the issue here if you or someone else have seen and resolved it by conifguration.

If I find the cause and fix I'll revert back.

`[2020-06-02T13:36:09.164Z] error Error: ﻿<?xml version="1.0" encoding="utf-8"?><Error><Code>BlobNotFound</Code><Message>The specified blob does not exist.
RequestId:67e613e9-601e-000b-32e2-38d5ba000000
Time:2020-06-02T13:36:09.1542884Z</Message></Error>
    at new RestError (/srv/strapi/stimulimanager/node_modules/@azure/ms-rest-js/dist/msRest.node.js:2433:28)
    at /srv/strapi/stimulimanager/node_modules/@azure/ms-rest-js/dist/msRest.node.js:3577:37
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Object.remove (/srv/strapi/stimulimanager/node_modules/strapi-plugin-upload/services/Upload.js:299:7)
    at async Object.destroy (/srv/strapi/stimulimanager/node_modules/strapi-plugin-upload/controllers/Upload.js:139:5)
    at async /srv/strapi/stimulimanager/node_modules/strapi/lib/middlewares/router/utils/routerChecker.js:70:22
    at async module.exports (/srv/strapi/stimulimanager/node_modules/strapi-plugin-users-permissions/config/policies/permissions.js:35:14)
    at async /srv/strapi/stimulimanager/node_modules/strapi-utils/lib/policy.js:52:5
    at async /srv/strapi/stimulimanager/node_modules/strapi/lib/middlewares/xss/index.js:26:9
[2020-06-02T13:36:09.165Z] debug DELETE /upload/files/1899 (85 ms) 500`

**Comment by alsmyr** [comment]
Found the cause. I had a / at the start of my defaultPath configuration parameter. Probably from an example somewhere. 

With **/static** for example the URL to the resource in uploading got https://XXX.blob.core.windows.net/stimuli//static/Sc... but the delete code removed the double // and thereby didn't find the resource to delete from Blob Storage.

My problem I fixed by changing the config. If you think it's a problem that this configuration is possible we could probably find a fix so that upload and delete are either both failing or both succeeding. 

Thanks' for a good tool!
